projects/example/src/app/app.component.ts

```diff
@@ -1,33 +1,40 @@
 import { Component, OnInit } from '@angular/core';
 import { FronteggAppService } from 'frontegg-app';
-import { BehaviorSubject } from 'rxjs';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss'],
 })
 export class AppComponent implements OnInit {
-  private fronteggAppStateSubject$ = new BehaviorSubject<any>(null);
+  private fronteggAppState: any;
```

Comment:
why the types is any?

Reply:
There aren't types for the state. We have any in interfaces